uint8 block       # Upper or lower block
uint16 encoder    # Encoder angle in 1/100th degree units.  0-35999 
Return[32] laser
geometry_msgs/Point32[32] point
float64 timestamp

uint8 BLOCK_UPPER = 0
uint8 BLOCK_LOWER = 1
uint8 NUM_BEAMS = 32
